### ISSUE TITLE
Blockquote added through the WYSIWYG has no default styling

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_base#611]
If you add a blockquote through the WYSIWYG no styling is applied.

To replicate

- with localgov_base as the default theme
- Add a text paragraph to a subsite page
- In the WYSYWYG add a quote - note the styling appears in the admin theme
- Save
- Notice no styling. 

The semantic styling should be replicated in the visual styling also.
